crate/memories/benches/benches.rs

```diff
@@ -30,7 +30,7 @@ fn get_redis_url() -> String {
 
 /// Refer to `src/memory/postgresql_store/memory.rs` for local setup instructions
```

Comment:
To be moved elsewhere as is it unclear what you are talking about. Or make it explicit.